lib/output.js

```diff
@@ -1185,6 +1200,21 @@ function OutputStream(options) {
         output.print(self.getValue());
     });
     DEFPRINT(AST_String, function(self, output){
+        var parent = output.parent(1);
```

Comment:
And strings are usually wrapped in AST_SimpleStatement. It's checking the parent of AST_SimpleStatement to see if that is a AST_Scope. If that's the case, it is probably a lonely string.